src/ResourceManagement/Sql/SqlDatabaseImpl.cs

```diff
@@ -159,6 +157,22 @@ public SqlDatabaseImpl WithEdition(string edition)
             return this;
         }
 
+        ///GENMHASH:376C162D9E950FA8A14B198790B45E34:5BF104F4489A5E42BAC27A2492EFC800
+        public IReadOnlyList<Microsoft.Azure.Management.Sql.Fluent.ISqlDatabaseUsageMetric> ListUsageMetrics()
```

Comment:
This method seems to be not implemented. Please check.

Reply:
fixed